homeassistant/components/ambee/sensor.py

```diff
@@ -62,26 +46,18 @@ def __init__(
         *,
         coordinator: DataUpdateCoordinator,
         entry_id: str,
-        sensor_key: str,
-        sensor: AmbeeSensor,
+        description: SensorEntityDescription,
         service_key: str,
         service: str,
     ) -> None:
         """Initialize Ambee sensor."""
         super().__init__(coordinator=coordinator)
-        self._sensor_key = sensor_key
+        self._sensor_key = description.key
```

Comment:
No need to keep this around, you can just use `self._attr_entity_description.key` in `AmbeeSensorEntity.state` instead of `self._sensor_key`

Reply:
Yup, indeed, will adjust when I am back at the office

Reply:
Adjusted.